configure.ac

```diff
@@ -5822,13 +5822,24 @@ AC_ARG_WITH(
     AS_CASE([$with_readline],
       [editline|edit], [with_readline=edit],
       [yes|readline], [with_readline=readline],
-      [no], [],
+      [no], [with_readline=default],
       [AC_MSG_ERROR([proper usage is --with(out)-readline@<:@=editline|readline|no@:>@])]
     )
   ],
-  [with_readline=readline]
+  [with_readline=default]
 )
 
+AS_VAR_IF([with_readline], [default], [
+  case $ac_sys_system/$ac_sys_release in
```

Comment:
Can you please use `AS_CASE` for new Autoconf code?

Reply:
> Can you please use AS_CASE for new Autoconf code?

Thanks, I am not expert for configure grammer, that's why I need you :)